### PR TITLE
Adds extended tooltip information to observables in the orbit ui

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
A continuation of #68389 which addresses an issue that still bothers me to this day:

The orbit menu displays a player's name as a combo of name id transform. It can get lengthy to a point where the names clip the entire screen (as buttons do not multiline).

This PR shortens excessively long player names on the orbit menu and adds a tooltip that will show extended info like full name, health and job titles.

Mostly drawn from concerns brought up in the original.

<details>
<summary>Screenshots</summary>
Health percentages, full names (what you currently see on live), and jobs

C5GcBXEgU8

Full disguises show up in buttons
rukdQmmh69

Some misc items get data **Suggestion via** @thebestcodertoevercode 
ytwj2WfAAt
Typo was nuked after this gif
</details>

## Why It's Good For The Game
Better observer experience
Bug fix (huge clipping names)

## Changelog

:cl:
add: The orbit UI now brings up tooltips to show health, job, and full name information.
add: The orbit UI now puts disguised names in quotes. "John Doe". Hover to get real name.
add: Orbit UI now gives tooltips on SM crystal, nuke disk, and nuclear devices.
add: You can now search by job in the orbit UI.
fix: Orbit UI buttons (should) no longer clip on extra long names.
/:cl:

### CODE REVIEW

**Comment by jlsnow301** [comment]
yes fyi I attempted to put icons in the tooltip because I wanted to recreate doomguy taking damage but alas

**Comment by Mothblocks** [comment]
can you be more specific in your title

File: tgui/packages/tgui/interfaces/Orbit.tsx [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+type Antags = Array<Observable & { antag: string }>;
 
 type Data = {
-  alive: Observable[];
-  antagonists: Observable[];
-  dead: Observable[];
-  ghosts: Observable[];
-  misc: Observable[];
-  npcs: Observable[];
+  alive: POIs;
```
Comment (Mothblocks) [inline tgui/packages/tgui/interfaces/Orbit.tsx:13]: can we not do this

Comment (Mothblocks) [reply tgui/packages/tgui/interfaces/Orbit.tsx:13]: POI is a bad technical term and Observable is what it literally is

Comment (jlsnow301) [reply tgui/packages/tgui/interfaces/Orbit.tsx]: Sorry I am trying to get into the habit of using the array<> syntax and it looked ugly

**Comment by thebestcodertoevercode** [comment]
Looks good, is it possible to add a dropdown box for the nuke to see if it's activated + a timer?

**Comment by jlsnow301** [comment]
> Looks good, is it possible to add a dropdown box for the nuke to see if it's activated + a timer?

This is a good idea, I will see if I can make it work

**Comment by jlsnow301** [comment]
Fixed an issue with critical not showing (oops, negative health) and disguises now show in a funnier way on the button
Screenshot 2022-10-15 181453

**Comment by jlsnow301** [comment]
> Looks good, is it possible to add a dropdown box for the nuke to see if it's activated + a timer?

This is done, but I should note that the orbit UI gets /static/ data. I made the tooltip a little more obvious that what you're seeing is stale. Reopening or refreshing the screen works.

**Review by tralezab — APPROVED** [review]

**Pull request merged**